econml/sklearn_extensions/model_selection.py

```diff
@@ -255,7 +260,112 @@ def get_n_splits(self, X, y, groups=None):
         """
         return self.n_splits
 
+class SearchEstimatorList(BaseEstimator):
+    def __init__(self, estimator_list = ['linear', 'forest'], param_grid_list = 'auto', scaling=True, is_discrete=False, scoring=None,
+                 n_jobs=None, refit=True, grid_folds=3, verbose=2, pre_dispatch='2*n_jobs', random_state=None,
+                 error_score=np.nan, return_train_score=False):
+        
+        self.estimator_list = estimator_list 
+        self.complete_estimator_list = get_complete_estimator_list(clone(estimator_list, safe=False), 'discrete' if is_discrete else 'continuous')
+
+        # TODO Add in more functionality by checking if it's an empty list. If it's just 1 dictionary then we're going to need to turn it into a list
+        # Just do more cases
+        if param_grid_list == 'auto':
+            self.param_grid_list = auto_hyperparameters(estimator_list=self.complete_estimator_list, is_discrete=is_discrete)
+        elif (param_grid_list == None):
+            self.param_grid_list = len(self.complete_estimator_list) * [{}]
+        else:
+            self.param_grid_list = param_grid_list
+        # self.categorical_indices = categorical_indices
```

Comment:
I think we do need to track categorical indices, since depending on the model we may need to one-hot-encode them and we also should exclude them from scaling.